### ISSUE TITLE
minimal sample would be useful

### ISSUE THREAD

**Issue by Gys** [volatiletech/authboss#244]
I am looking into authboss now to see if its usable for a new project. So far I think this really overwhelming. Many settings, many packages. It would be useful to see some minimal example program which is easily to download and get running. Then go from there. So something without blogs, with 2fa, without API (json). Just register, login, logout. The existing full example will be very useful to study for adding elements as a next step.

**Comment by frederikhors** [comment]
You don't like https://github.com/volatiletech/authboss-sample?

**Comment by Gys** [comment]
Yes, I have seen authboss-sample and read parts of the documentation. I have my own implementation running now. Its just that normally a minimal version is easier to start with then a full version like authboss-sample.
